File: android/src/main/java/com/yugioh/android/CoinDiceActivity.java
package com.yugioh.android;

import android.app.Fragment;

import com.yugioh.android.base.BaseActivity;
import com.yugioh.android.fragments.CoinDiceFragment;

public class CoinDiceActivity extends BaseActivity {

	@Override
	public int getIcon() {
		return R.drawable.icon;
	}

	@Override
	public Fragment replaceFragment() {
		return new CoinDiceFragment();
	}

    @Override
    public int customTheme() {
        return 0;
    }

}

File: android/src/main/kotlin/com/yugioh/android/CoinDiceActivity.kt
package com.yugioh.android

import android.app.Fragment
import com.rarnu.base.app.BaseActivity
import com.yugioh.android.fragments.CoinDiceFragment

class CoinDiceActivity : BaseActivity() {

    override fun getActionBarCanBack(): Boolean = true

    override fun getIcon(): Int = R.drawable.icon

    override fun replaceFragment(): Fragment = CoinDiceFragment()

    override fun customTheme(): Int = 0

}
